src/BTVNanoCommissioning/workflows/ttsemilep_valid_sf.py

```diff
@@ -176,8 +211,8 @@ def process(self, events):
         event_muon = ak.pad_none(events.Muon, 1, axis=1)
 
         req_muon = ak.count(event_muon.pt, axis=1) == 1
-        ## Jet cuts
 
+        ## Jet cuts
```

Comment:
Is there a reason why no jet puID requirements are applied for the semileptonic ttbar SF workflow, but for the dileptonic ttbar SF workflow?

Reply:
ah, indeed it should applied in both workflow. add in next commit

Reply:
All the workflows contain puId requirements
https://github.com/cms-btv-pog/BTVNanoCommissioning/blob/3c0b4cdf230215dab8b53ca62797fe5037026d82/src/BTVNanoCommissioning/workflows/ttsemilep_valid_sf.py#L245

Reply:
Apologies, I failed to spot the PUID requirement during review